parser.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python3
+# parser.py
+
+
+def parse(text):
+    lines = str(text).splitlines()
+    while("" in lines):
+        lines.remove("")
+    words = []
+    for l in lines:
+        row = str(l).split(" ")
+        while("" in row):
+            row.remove("")
+        word = dict()
+        word["head"] = int(row[0])
```

Comment:
Can you change these names to match [the specification](http://universaldependencies.org/format.html)? The most confusing bit is that you've switched "head" and "id" :P but fix the other two too. Also note that the "id" isn't always an integer, it can be a decimal or a range, but the "head" will always point to an integer "id".